docs/source/EquivalenceChecking.rst

```diff
@@ -77,7 +77,7 @@ ZX-Calculus Equivalence Checker
 In :cite:p:`kissinger2019PyZXLargeScale`, Kissinger and van de Wetering proposed an algorithm for optimizing quantum circuits based on ZX-calculus rewriting.
 In their initial article they also show that this rewriting approach can be used to prove the equivalence of quantum circuits.
 The idea is to construct the ZX-diagram of :math:`G^{\prime -1} G`  and reduce this ZX-diagram using the rules of the ZX-calculus until no further rewrites can be made.
-If the resulting diagram consists only of wires (the identity ZX-diagram) then :math:`G = G^{\prime}`.
+If the resulting diagram consists only of wires (the identity ZX-diagram) then :math:`G = G^{\prime}`. This approach has been extended in :cite:p:`peham2022equivalenceCheckingZXCalculus` to also handle ancillary qubits, floating point inaccuracies in gate parameters, as well as permutations of input and output layout of quantum circuits.
 
 In :cite:p:`peham2022EquivalenceCheckingParadigms`, it has been shown that equivalence checking with the ZX-calculus naturally complements equivalence checking with decision diagrams.
```

Comment:
Doesn't the JETCAS paper supersede the DAC paper? I would have probably replaced this citation with the JETCAS paper as well. On the other hand, the DAC paper wouldn't be cited anywhere in the documentation otherwise.
I'll leave it to you whether you want to cite both papers here.

Reply:
You are right about the JETCAS Paper superseding the DAC one. But since the complementarity of DDs and ZX was the main message of the DAC Paper and not the main focus of the JETCAS Paper I would leave it.